README.md

```diff
@@ -0,0 +1,136 @@
+├── base
+│   └── ubi8-python-3.8
+│       └── Dockerfile		#Includes the instructions for the base image
+├── LICENSE
+├── Makefile                #Lists shell commands for the build and push process
+├── OWNERS                  #Includes the mainteiners of the repository
+└── README.md               #Includes basic information about the repository
+
+```
+
+#### Container Image Layering
+In order to avoid packages conflicts and improve the performance of the building process, the notebooks at the bottom are built using the previous notebook as their base image:
+
+```mermaid
+flowchart TD
+    ubi/python-3.8 --> python-3.8-minimal;
+    python-3.8-minimal --> python-3.8-standar-data-science;
+    python-3.8-minimal --> python-3.8-cuda;
+    python-3.8-cuda --> python-3.8-cuda-pytorch;
+    python-3.8-cuda --> python-3.8-cuda-tensorflow;
+```    
+
+
+The notebook images name will have the following format:
+```${UBI_VERSION}-${KERNEL_LANG}-${LANG_VERSION}-${NOTEBOOK_ID}```
+
+
+**Build Command for the Base Image**
+
+```make base-ubi8-python-3.8```
+
+##
+
+### Python Minimal
+
+
+Use the Red Hat UBI image as the base image of the notebook:
+
+-   Base image for Python 3.8 will be [registry.access.redhat.com/ubi8/python-38](https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a)
+    
+
+Install the required packages* to run a Jupyterlab notebook server:
+
+-   Python 3.8
+    
+-   JupyterLab 3.2
+    
+-   Notebook 6.4
+    
+
+*Note: These are the initial packages that will be shipped in the first release of the notebooks as stated in the [RHODS documentation](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/getting_started_with_red_hat_openshift_data_science/launching-jupyterhub-and-starting-a-notebook-server_get-started#options-for-notebook-server-environments_get-started), packages will be upgraded according to [Updating Notebooks](https://docs.google.com/document/d/1H6KUvY80LfZPBNYBMT-amILAkZ75P5LaSYE9rCerqxw/edit#heading=h.7nhjrey44j2) procedure.
+
+### Python Standard Data Science
+
+Extend the minimal notebook with commonly used libraries to assist customers in the development of machine learning models:
+
+-   Boto3 1.17
+    
+-   Kafka-Python 2.0
+    
+-   Matplotlib 3.4
+    
+-   Pandas 1.2
+    
+-   Numpy 1.19
+    
+-   Scikit-learn 0.24
+    
+-   SciPy 1.6
+### Python CUDA PyTorch
+
+Extend the CUDA notebook with the PyTorch packages:
+
+-   PyTorch 1.8
+    
+-   TensorBoard 1.15
+    
+-   Boto3 1.17
+    
+-   Kafka-Python 2.0
+    
+-   Matplotlib 3.4
+    
+-   Numpy 1.19
+    
+-   Pandas 1.2
+    
+-   Scikit-learn 0.24
+    
+-   SciPy 1.6
+    
+
+### Python CUDA TensorFlow
+
+Extend the CUDA notebook with the TensorFlow packages:
+
+-   TensorFlow 2.7
+    
+-   TensorBoard 2.6
+    
+-   Boto3 1.17
+    
+-   Kafka-Python 2.0
+    
+-   Matplotlib 3.4
+    
+-   Numpy 1.19
+    
+-   Pandas 1.2
+    
+-   Scikit-learn 0.24
+    
+-   SciPy 1.6
+
```

Comment:
We don't need all this information in the README. Instead we need a section per image with a **brief** explanation of the image and the instructions to manually build the image. Add only the base image instructions for now.